aspnetcore/tutorials/signalr.md

```diff
@@ -90,76 +87,85 @@ At the end, you'll have a working chat app:
 
 ## Add the SignalR client library
 
-The SignalR server library is included in the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app). But you have to get the JavaScript client library from [npm, the Node.js package manager](https://www.npmjs.com/get-npm).
+The SignalR server library is included in the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app). The JavaScript client library isn't automatically included in the project. For this tutorial, you use the [Library Manager (LibMan)](xref:client-side/libman/index) to get the client libary from *unpkg*. [unpkg](https://unpkg.com/#/) is a [content delivery network](https://en.wikipedia.org/wiki/Content_delivery_network) that can deliver anything found in [npm, the Node.js package manager](https://www.npmjs.com/get-npm).
 
 # [Visual Studio](#tab/visual-studio/)
 
-* In **Package Manager Console** (PMC), change to the project folder (the one that contains the *SignalRChat.csproj* file).
+* In **Solution Explorer**, right-click the project, and then select **Add > Client-Side Library**.
 
-  ```console
-  cd SignalRChat
-  ```
+* In the **Add Client-Side Library** dialog, for **Provider** select **unpkg**. 
 
-# [Visual Studio Code](#tab/visual-studio-code/)
+* For **Library**, enter _@aspnet/signalr@1_, and then select the latest version that isn't preview.
 
-2. Change to the new project folder.
+  ![Add Client-Side Library dialog - select library](signalr/_static/libman1.png)
 
-  ```console
-  cd SignalRChat
-  ``` 
+* Select **Choose specific files**, expand the *dist/browser* folder, and then select *signalr.js* and *signalr.min.js*.
 
-# [Visual Studio for Mac](#tab/visual-studio-mac)
+* Set **Target Location** to *wwwroot/lib/signalr/*, and then select **Install**.
 
-* In the **Terminal**, navigate to the project folder (the one that contains the *SignalRChat.csproj* file).
+  ![Add Client-Side Library dialog - select files and destination](signalr/_static/libman2.png)
 
----
+  [LibMan](xref:client-side/libman/index) creates a *wwwroot/lib/signalr* folder and copies the selected files to it.
+
+# [Visual Studio Code](#tab/visual-studio-code/)
 
-* Run the npm initializer to create a *package.json* file:
+* In the **Integrated Terminal**, navigate to the project folder (the one that contains the *SignalRChat.csproj* file).
```

Comment:
The `dotnet tool install` command can be run from anywhere. Consider moving the "navigate to the project folder" bit to the next step, since that's where it's required.